### ISSUE TITLE
Decreased circulating insulin-like growth factor binding protein 2 concentration

### ISSUE THREAD

**Issue by carolina-f** [obophenotype/human-phenotype-ontology#11597]
**Preferred term label:**

**Synonyms**
Reduced serum IGFBP-2 level
Reduced IGF binding protein-2 level

**Definition (free text, please give PubMed ID)**
PMID: 19729943 Human ALS deficiency results in a peculiar form of IGF-I deficiency. Whereas circulating levels of IGF-I decrease dramatically, local production, and presumably action, appears to be preserved. In addition to IGF-I, other members of the circulating IGF system are also affected. Circulating IGF-II levels are reduced , while IGFBP-3 levels are even more profoundly diminished. **Levels of IGFBP-1 and -2 are reduced, although to a lesser extent**.

PMID: 21396577 The absence of ALS in the circulation resulted in the generalized deficiency of the circulating, so called “endocrine IGF system,” with reduction of total IGF-I and IGF-II, as well as free-IGF-I and bioactive IGF-I. While IGFBP-3 levels were even more profoundly reduced, **IGFBP-1 and-2 were found to be also reduced, although to a lesser extent**

**Parent term (use hpo.jax.org/app)**
Abnormal circulating hormone concentration  HP:0003117

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
Short stature due to primary acid-labile subunit deficiency ORPHA: 140941

**Your nano-attribution (ORCID)**
0009-0005-6714-5727
